civihr_employee_portal/templates/civihr-employee-portal-civihr-report-settings-age-group.tpl.php

```diff
@@ -6,7 +6,7 @@
         <span class="chr_search-result__total__count"><?php echo count($rows); ?></span>
       </div>
     </div>
-
+    <?php $classes .= ' table '; ?>
```

Comment:
I discussed with @coldrunKacper and it is not a problem to addd it here because it is a custom template.